staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2446,6 +2446,24 @@ type Container struct {
 	// +optional
 	// +listType=atomic
 	ResizePolicy []ContainerResizePolicy `json:"resizePolicy,omitempty" protobuf:"bytes,23,rep,name=resizePolicy"`
+	// Restart policy for the container to manage the restart behavior of each
+	// container within a pod.
+	// By default, if no value is assigned to this field, the restart behavior of
+	// the container follows the Pod's RestartPolicy. This field may only be set
+	// for init containers and only allowed value is "Always".
+	// "Always" results in this init container being continually restarted on
+	// exit until all regular containers have terminated. Once all regular
+	// containers have terminated and not being restarted, the init container
+	// with restartPolicy "Always" will be shut down. This lifecycle differs from
+	// normal init containers and is often referred to as a "sidecar" container.
+	// Although this init container still starts in the init container sequence,
+	// it does not wait for the container to complete before proceeding to the
+	// next init container. Instead, the next init container starts immediately
+	// after this init container has begun, and its startupProbe and postStart
```

Comment:
`immediately after this init container is started, or after any startupProbe and postStart hooks have successfully completed.`